sdk/nodejs/policy/server.ts

```diff
@@ -87,9 +94,9 @@ export function serve(policyPackName: string, policyPackVersion: string, policie
     // Finally connect up the gRPC client/server and listen for incoming requests.
     const server = new grpc.Server();
     server.addService(analyzerrpc.AnalyzerService, {
-        analyze: makeAnalyzeRpcFun(policyPackName, policyPackVersion, policies),
-        analyzeStack: makeAnalyzeStackRpcFun(policyPackName, policyPackVersion, policies),
-        getAnalyzerInfo: makeGetAnalyzerInfoRpcFun(policyPackName, policyPackVersion, policies),
+        analyze: makeAnalyzeRpcFun(policyPackName, policyPackVersion, policyPackEnforcementLevel, policies),
```

Comment:
Hmm instead of constantly changing this interface, I am wondering if it makes sense to make a PolicyPack type 🤔